test/integrationtests/LorisIntegrationTest.class.inc

```diff
@@ -287,8 +287,8 @@ abstract class LorisIntegrationTest extends PHPUnit_Framework_TestCase
      */
     function changeStudySite()
```

Comment:
This test function should take psc and users as parameters instead of hardcoded.  Will enable others to use this function, testing for different sites' access, and for user profiles with different permissions.